Firestore/Source/Core/FSTSyncEngine.mm

```diff
@@ -523,15 +524,14 @@ - (void)emitNewSnapshotsAndNotifyLocalStoreWithChanges:(const MaybeDocumentMap &
 
         if (viewChange.snapshot.has_value()) {
           newSnapshots.push_back(viewChange.snapshot.value());
-          FSTLocalViewChanges *docChanges =
-              [FSTLocalViewChanges changesForViewSnapshot:viewChange.snapshot.value()
-                                             withTargetID:queryView.targetID];
-          [documentChangesInAllViews addObject:docChanges];
+          LocalViewChanges docChanges =
+              LocalViewChanges::MakeChanges(viewChange.snapshot.value(), queryView.targetID);
+          documentChangesInAllViews.push_back(docChanges);
         }
       }];
 
   [self.syncEngineDelegate handleViewSnapshots:std::move(newSnapshots)];
-  [self.localStore notifyLocalViewChanges:documentChangesInAllViews];
+  [self.localStore notifyLocalViewChanges:std::move(documentChangesInAllViews)];
```

Comment:
No need to move here (after you change the signature of `notifyLocalViewChanges`).

Reply:
Done.